source/includes/usage-examples/read-write-pref.rb

```diff
@@ -0,0 +1,80 @@
+require 'bundler/inline'
+
+gemfile do
+  source 'https://rubygems.org'
+  gem 'mongo'
+end
+
+uri = '<connection string>'
+
+Mongo::Client.new(uri) do |client|
+    # Access the database and collection
```

Comment:
I know this script isn't intended to be run, but the four-space indent here makes my eye-twitch. :) We prefer 2-space indents on the Ruby team. This block is also missing an `end` keyword.